### ISSUE TITLE
Import dashboard fails for beats context deadline exceeded

### ISSUE THREAD

**Issue by liza-mae** [elastic/beats#27586]
For confirmed bugs, please report:
- Version: 7.15, 7.16, 8.0 
- Operating System:  Ubuntu 
- Steps to Reproduce: 
```./filebeat setup```

```
01:16:46 Exiting: Failed to import dashboard: Failed to load directory /home/vagrant/filebeat-7.15.0-SNAPSHOT-linux-x86_64/kibana/7:
01:16:46   error loading /home/vagrant/filebeat-7.15.0-SNAPSHOT-linux-x86_64/kibana/7/dashboard/Filebeat-threatintel-recordedfuture.json: error dashboard asset: returned 0 to import file: fail to execute the HTTP POST request: Post "https://10.0.2.15:5601/api/saved_objects/_import?overwrite=true": context deadline exceeded (Client.Timeout exceeded while awaiting headers). Response: 
01:16:46   error loading /home/vagrant/filebeat-7.15.0-SNAPSHOT-linux-x86_64/kibana/7/dashboard/filebeat-network-flows-top-n.json: error dashboard asset: returned 0 to import file: fail to execute the HTTP POST request: Post "https://10.0.2.15:5601/api/saved_objects/_import?overwrite=true": context deadline exceeded (Client.Timeout exceeded while awaiting headers). Response: 
```

**Comment by liza-mae** [comment]
Now fails with 
```
error loading /home/vagrant/filebeat-8.0.0-SNAPSHOT-linux-x86_64/kibana/7/dashboard/filebeat-network-flows-top-n.json: error dashboard asset: returned 0 to import file: fail to execute the HTTP POST request: Post "https://10.0.2.15:5601/api/saved_objects/_import?overwrite=true": context deadline exceeded 
```

**Comment by liza-mae** [comment]
@kvch this is now broken for 7.15, needs to be fixed before release. 
Is it related to this PR: https://github.com/elastic/beats/pull/27647 ?

**Comment by kvch** [comment]
It is related. I have a several open PRs in progress: https://github.com/elastic/beats/pull/27749 https://github.com/elastic/beats/pull/27743

**Comment by kvch** [comment]
The PRs are merged. The error should be gone now.

**Comment by liza-mae** [comment]
This is still failing on 7.15, it contains #27743, does it also need #27749 back ported?

**Comment by liza-mae** [comment]
On 7.15
```
22:17:30 Exiting: Failed to import dashboard: Failed to load directory /home/vagrant/metricbeat-7.15.0-linux-x86_64/kibana/7:
22:17:30   error loading /home/vagrant/metricbeat-7.15.0-linux-x86_64/kibana/7/dashboard/Metricbeat-aws-kinesis-overview.json: error dashboard asset: returned 0 to import file: fail to execute the HTTP POST request: Post "https://10.0.2.15:5601/api/saved_objects/_import?overwrite=true": context deadline exceeded (Client.Timeout exceeded while awaiting headers). Response: 
```

**Comment by liza-mae** [comment]
We only have one more BC scheduled, so we need this fixed for 7.15.

@kvch, can you shed some light on the new variation of the same error?

**Comment by kvch** [comment]
The fix have been merged in https://github.com/elastic/beats/pull/27784 on Monday. Based on the git hash in the output of `./metrcibeat version` this patch is not included.
